app/src/main/java/com/adt/game_of_life/model/algorithm/TemporaryAlgorithmInitializer.kt

```diff
@@ -0,0 +1,54 @@
+package com.adt.game_of_life.model.algorithm
+
+//TODO: Klasa do usuniecia po dodaniu klasy algorytmu w poprawne miejsce aplikacji
+//Wystaczy wywolac makeGameStep() - otrzymujemy numer kroku i wyglad planszy w consoli
+
```

Comment:
Fajnie by było mergować z już poprawnie wrzuconymi klasami w odpowiednie miejsca.

Reply:
Z tego co kojarzę to jest wrzucone w dobre miejsce.

Reply:
W sumie na przyszłość fajnie by było gdyby ConwayAlgorithm  implementował jakiś interfejs np IConwayAlgorithm. Wtedy moduły byłyby luźniej powiązane i moglibyśmy użyć wstrzykiwania zależności.

Reply:
> Z tego co kojarzę to jest wrzucone w dobre miejsce.

Jeśli tak to bez mylących komentarzy, szczególnie z TODO.